Packs/ThreatConnect/Integrations/ThreatConnectV3/ThreatConnectV3.py

```diff
@@ -30,12 +55,12 @@ def __init__(self, api_id: str, api_secret: str, base_url: str, verify: bool = T
         self.base_url = base_url
         self.verify = verify
 
-    def make_request(self, method: Method, url_suffix: str, payload: dict = None, params: dict = None,
+    def make_request(self, method: Method, url_suffix: str, payload: str = None, params: dict = None,
```

Comment:
is there a reason why to define `payload` as `str` and then using all the time `json.dumps` as the `data` in `_http_request` defined as `dict`?

Reply:
The API supports only str objects. In the original implementation we used `json.dumps` in all request and put the `type: ignore` on all of them. I thought it would be better to put the ignore in one place